### ISSUE TITLE
HTTP 401 Error

### ISSUE THREAD

**Issue by Artemaius** [floccusaddon/floccus#1623]
### Which version of floccus are you using?

5.1.4

### How many bookmarks do you have, roughly?

200

### Are you using other means to sync bookmarks in parallel to floccus?

No

### Sync method

Git

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Librewolf 126.0-1

### Describe the Bug

HTTP Error 401 when synchronizing, tried using both email and github username, same error.
[floccus-5.1.4-2024-05-24-redacted.log](https://github.com/floccusaddon/floccus/files/15428873/floccus-5.1.4-2024-05-24-redacted.log)

### Expected Behavior

Sync should happen without errors

### To Reproduce

1. Create profile
2. Select Git
3. Paste repository link (Starting with https and ending with .git)
4. Select bookmarks toolbar as local target
5. Default settings for the rest
6. Sync fails

### Debug log provided

- [X] I have provided a debug log file

**Comment by marcelklehr** [comment]
If you get a 401 your authentication credentials are likely wrong.

**Comment by Artemaius** [comment]
That was it, I was using Github so a personal access token is needed, thank you

**Comment by Micemade** [comment]
Hi.
I have a similar problem. The authentication with my GitHub account password didn't work, so I set up a Personal Access Token, granted permission, but still cannot sync. This is the error notice:

`E034: Remote bookmarks file is unreadable. Perhaps you forgot to set an encryption passphrase, or you set the wrong file format. | Last synchronized: 0 seconds ago `

The file format is bookmarks.xbel, and I don't understand this about the encryption passphrase.

**Comment by marcelklehr** [comment]
Hello @Micemade 
Could you please open a new issue for this problem, so I can try to help you properly?
